spec/models/application_spec.rb

```diff
@@ -2,7 +2,17 @@
 
 RSpec.describe Application, type: :model do 
   describe 'relationships' do 
-    it { should have_many :application_pets }
-    it { should have_many(:pets).through(:application_pets) }
+    it {should have_many :application_pets}
+    it {should have_many(:pets).through(:application_pets)}
+  end
+
+  describe 'validations' do 
```

Comment:
I trust you on this, but I don't fully understand what this is solving. Can we follow up on this irl?